modules/docs/mdx/7.0-UPGRADE-GUIDE.mdx

```diff
@@ -377,41 +377,39 @@ We've also changed the default depth for the following components:
 
 ### Component Deprecations
 
-#### Deprecation Types
+### Deprecation Types
 
-There are two types of deprecations: soft and hard.
+There is `deprecated` and `removal`.
 
-##### Soft Deprecation
+#### Deprecation
 
-A soft-deprecated component is still available with its full functionality, but it will have been
-renamed with a prefix to indicate its soft-deprecated status. It will also include a console warning
-announcing its deprecation. This warning will only be triggered on the component's initial render.
+We add the [@deprecated](https://jsdoc.app/tags-deprecated.html) from JSDoc to code that we plan to
+deprecate in the near future. The component name will also have a strikethrough to indicate its
+deprecated status. Although you can still consume this code, we want consumers to move to a utility
+or component that is more stable.
 
-Soft-deprecated types and utilities will also be renamed but generally will not trigger a console
-warning.
+#### Removal
 
-##### Hard Deprecation
-
-A hard-deprecated component or package is no longer available. You will need to follow the method
-prescribed in our upgrade guide to update your application. Please
-[reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) to our team
-directly if you need additional help.
+Removals are deletions from our codebase and you can no longer consume this component. We've either
+promoted or replaced a component or utility. You will need to follow the method prescribed in our
+upgrade guide to update your application. Please reach out to our team directly if you need
+additional help.
 
 #### Cookie Banner
 
-We are [hard-deprecating](#hard-deprecation) `CookieBanner`. Please reference this
+We are [deprecating](#deprecation) `CookieBanner`. Please reference this
```

Comment:
I think where we have "hard-deprecating" it should be removing?

Reply:
Correct and has been updated